bookmarket-backend/src/main/java/edu/oakland/sophomoreproject/controllers/ListingsController.java

```diff
@@ -79,7 +79,13 @@ public ResponseEntity<Void> createListing(
 			return ResponseEntity.status(403).build();
 		}
 
-		// ... do logic here
+		String title = payload.getTitle();
+		String author = session.getEmail(); // Just used email for now can change to name or something else if needed.
+		String classSubject = payload.getClassSubject();
+		String description = payload.getDescription();
+		String condition = payload.getCondition();
+		String saleAvailability = payload.getSaleAvailability();
+		int sellerId = session.getUserId();
 
```

Comment:
You need to now actually call `listingsTableAccessor.createListing(...)` in the controller, right now it is not doing anything